frontend/package.json

```diff
@@ -9,8 +9,10 @@
     "axios": "^0.20.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-icons": "^4.1.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.3"
+    "react-scripts": "3.4.3",
```

Comment:
why need of this package ?

Reply:
@ashishnagpal2498 react-scripts is in the original package.json file. I haven't added that.